nexus/reconfigurator/planning/tests/output/planner_nonprovisionable_2_2a.txt

```diff
@@ -0,0 +1,106 @@
+-    sled 68d24ac5-f341-49ea-a92a-0381b52ab387: zones at generation 2 (removed)                                 
+                                                                                                                
+-        01d58626-e1b0-480f-96be-ac784863c7dc        in service          nexus         fd00:1122:3344:102::22   
+-        3b3c14b6-a8e2-4054-a577-8d96cb576230        in service        crucible        fd00:1122:3344:102::2c   
+-        47a87c6e-ef45-4d52-9a3e-69cdd96737cc        in service        crucible        fd00:1122:3344:102::23   
+-        6464d025-4652-4948-919e-740bec5699b1        in service        crucible        fd00:1122:3344:102::24   
+-        6939ce48-b17c-4616-b176-8a419a7697be        in service        crucible        fd00:1122:3344:102::29   
+-        878dfddd-3113-4197-a3ea-e0d4dbe9b476        in service        crucible        fd00:1122:3344:102::25   
+-        8d4d2b28-82bb-4e36-80da-1408d8c35d82        in service        crucible        fd00:1122:3344:102::2b   
+-        9fd52961-426f-4e62-a644-b70871103fca        in service        crucible        fd00:1122:3344:102::26   
+-        b44cdbc0-0ce0-46eb-8b21-a09e113aa1d0        in service        crucible        fd00:1122:3344:102::27   
+-        b6b759d0-f60d-42b7-bbbc-9d61c9e895a9        in service        crucible        fd00:1122:3344:102::28   
+-        c407795c-6c8b-428e-8ab8-b962913c447f        in service        crucible        fd00:1122:3344:102::2a   
+-        f3f2e4f3-0985-4ef6-8336-ce479382d05d        in service      internal_ntp      fd00:1122:3344:102::21   
+                                                                                                                
+     sled 2d1cb4f2-cf44-40fc-b118-85036eb732a9: zones at generation 2                                           
+                                                                                                                
+~      changed zones:                                                                                           
+~        19fbc4f8-a683-4f22-8f5a-e74782b935be  :  policy        in service             -> not in service        
+                                               :  zone type     crucible                  (unchanged)           
+                                               :  underlay IP   fd00:1122:3344:105::26    (unchanged)           
+~        6dff7633-66bb-4924-a6ff-2c896e66964b  :  policy        in service                (unchanged)           
+                                               :  zone type     nexus                     (config changed)      
+                                               :  underlay IP   fd00:1122:3344:105::22    (unchanged)           
+~        7f4e9f9f-08f8-4d14-885d-e977c05525ad  :  policy        in service                (unchanged)           
+                                               :  zone type     internal_ntp              (unchanged)           
```

Comment:
I don't think the zone type can ever change (I think we should mandate that)

Reply:
Done by eagerly computing the diff and returning an error if the zone type changes.